libs/platform/src/lib/components/page-footer/page-footer.component.scss

```diff
@@ -0,0 +1,268 @@
+            color: $fd-copy-right-font-color;
+            font-family: var(--sapFontFamily, '72', '72full', Arial, Helvetica, sans-serif);
+            font-size: 0.875rem;
+            text-align: right;
+            display: -webkit-box;
+            width: 30%;
+            position: absolute;
+            bottom: 0;
+            right: 0;
+            overflow: hidden;
+            -webkit-line-clamp: 3;
+            -webkit-box-orient: vertical;
+            @include fd-rtl() {
+                text-align: left;
+                left: 0;
+                right: auto;
+            }
+        }
+    }
+}
+
+@media (min-width: 361px) and (max-width: 768px) {
+    .fdp-footer {
+        @include fd-reset();
+
+        &__container {
+            width: 100%;
+        }
+
+        width: 100%;
+        background-color: #fefefe;
+        display: inline-block;
+        padding: 1.5rem 1rem 2rem 1rem;
+
+        &__logo {
+            width: 3rem;
+            height: 1.5rem;
+            overflow: hidden;
+        }
+        &__link {
+            color: $fd-link-font-color;
+            font-family: var(--sapFontFamily, '72', '72full', Arial, Helvetica, sans-serif);
+            font-size: 0.875rem;
+            display: inline-block;
+            padding-top: $fd-footer-padding-top;
+            padding-bottom: $fd-footer-padding-top;
+            > div > * {
+                margin-right: 1.5rem;
+                @include fd-rtl() {
+                    margin-left: 1.5rem;
+                    margin-right: 0rem;
+                }
+            }
+            > * {
+                margin-right: 1.5rem;
+                padding-top: 0.25rem;
+                @include fd-rtl() {
+                    margin-left: 1.5rem;
+                    margin-right: 0rem;
+                }
+            }
+        }
+        &__copy-right {
+            color: $fd-copy-right-font-color;
+            font-family: var(--sapFontFamily, '72', '72full', Arial, Helvetica, sans-serif);
+            font-size: 0.875rem;
+            display: -webkit-box;
+            overflow: hidden;
+            -webkit-line-clamp: 3;
+            -webkit-box-orient: vertical;
+            @include fd-rtl() {
+                text-align: left;
+            }
+        }
+    }
+}
+@media (max-width: 360px) {
+    .fdp-footer {
+        @include fd-reset();
+
+        &__container {
+            width: 100%;
+            padding-top: $fd-footer-padding-top;
+        }
+
+        width: 100%;
+        background-color: #fefefe;
+        display: inline-block;
+        line-height: 1.25rem;
+        padding: 1.5rem 0.5rem 2rem 0.5rem;
+
+        &__logo {
+            width: 3rem;
+            height: 1.5rem;
+            overflow: hidden;
+        }
+        &__link {
+            color: $fd-link-font-color;
+            font-family: var(--sapFontFamily, '72', '72full', Arial, Helvetica, sans-serif);
+            font-size: 0.875rem;
+            display: inline-grid;
```

Comment:
For what we have `display: inline-grid;` here?

Reply:
second example distorts in the mobile view so needed.